quilter.py

```diff
@@ -0,0 +1,360 @@
+'''
+quilter.py
+
+Download ALL THE RASTERS!!!
+
+'''
+
+import csv
+import os
+import tempfile
+import zipfile
+import shutil
+import sys
+import requests
+import traceback
+import inspect
+import subprocess
+import argparse
+from pathlib import Path
+
+# User Input:
+# csv file
+# destination folder
+# -m merge flag and name
+# -p project flag and accompanying EPSG:xxxxx designator
+# -d don't compress tif (ignore for aerial imagery)
+
+parser = argparse.ArgumentParser(description='Download helper for\
+                                              raster.utah.gov')
+
+parser.add_argument('csv', help='CSV file of download links.')
+parser.add_argument('destination', 
+                    help='Directory for extracted files (must already exist).')
+
+parser.add_argument('-m', '--merge', dest="name", 
+                    help='Merge downloaded files to specified base name. .tif\
+                    or .shp extensions will be added as appropriate. Requires\
+                    GDAL.')
+parser.add_argument('-p', '--project', dest="crs", help='Reproject merged file\
+                    to specified CRS. Specify CRS like EPSG:x or ESRI:x.\
+                    Requires -m.')
+parser.add_argument('-d', '--dont_compress', help='Disable LZW compression of\
+                    merged raster (all aerials will remain compressed).',
+                    action='store_true')
+
+args = parser.parse_args()
+
+# Set defaults
+merge = False
+project = False
+compress = True
+delete = False
+
+outfolder = args.destination
+csv_file = args.csv
+if args.name:
```

Comment:
Will you add something to these ifs that tells the user how this is expected to work if they do not add any parameters or they add a `--help` or `-h` command?

Reply:
I'm not understanding what you're asking. Are you looking for something more than argparse's automatic `-h` generator and error message if called w/o arguments?

Reply:
I didn't realize it did that. I thought you had to be explicit about it.